sql/catalyst/src/main/antlr3/org/apache/spark/sql/catalyst/parser/SparkSqlLexer.g

```diff
@@ -465,7 +467,7 @@ Identifier
 fragment    
 QuotedIdentifier 
     :
-    '`'  ( '``' | ~('`') )* '`' { setText(getText().substring(1, getText().length() -1 ).replaceAll("``", "`")); }
+    '`'  ( '``' | ~('`') )* '`' { setText(getText().replaceAll("``", "`")); }
```

Comment:
`QuotedIdentifier` originally strips first and last backticks. So we don't know if an identifier is quoted or not. In create table ddl commands, you can specify a column name with quoted string to use special characters. If we strip backticks early at `QuotedIdentifier`, then we can not have these column names passed in. So I change it to not strip the first and last backticks and use `cleanIdentifier` to get rid of the backticks.